cmd/sidecars/BUILD.bazel

```diff
@@ -40,5 +40,8 @@ container_image(
     directory = "/",
     entrypoint = ["/sidecar-shim"],
     files = [":sidecar-shim"],
+    tars = [
+        "//rpm:generic-sidecar",
```

Comment:
We need to select based on arch as `sidecar-shim` image [supports](https://quay.io/repository/kubevirt/sidecar-shim?tab=tags) arm64 too. You can look at the [pr-helper's rpm](https://github.com/kubevirt/kubevirt/blob/main/cmd/pr-helper/BUILD.bazel#L21) as an example

Reply:
Done.